deploy/quickstart.sh

```diff
@@ -7,6 +7,7 @@ set -e
 #  Upstream containers: ./quickstart.sh
 #  Downstream containers: ./quickstart.sh --downstream-secret=~/6340056-cloudops-pull-secret.yaml
 oc login -u system:admin
+oc delete project sa-telemetry || true
```

Comment:
Hmmm, now this script is dangerous! 👎 

I don't like this, but I'd like it more if it were behind a flag like `./quickstart.sh --delete` so at least it would have to be deliberate and involve the word 'delete'. See L47 for an example of minimal command line parsing.

It's not significantly better than `oc delete project sa-telemetry; ./quickstart.sh`, though, so I'm not 100% sure I understand the workflow you're trying to solve for with these changes. I **think** the goal is _"I want the ability re-run the script quickly"_ (even quicker than a `oc delete project` if I understood your original comments). My objection to your method (`|| true`) is that it would ignore a failure to create the project, and then start building resources in some other namespace. In the other cases (L21 and L35), it would trigger a failure later on with SSL config which would be harder to debug then a failure here. So I guess what you're really looking for here is a new-project-if-not-exist. You could probably achieve that with `oc get project | grep sa-telemetry` and an if statement. See below for a slicker way to do this for other resources.

Reply:
If you want to speed things up, then perhaps just create an alias to the command?

`alias ocdp='oc delete project'`

Reply:
I will just have it check if the project already exists before having it try to build a new one